shortfin/python/shortfin_apps/llm/components/generate.py

```diff
@@ -54,10 +54,12 @@ async def run(self):
             token = sfnp.argmax(exec.result_logits)
             token_int = token.items[0]
 
+            self.append_token(token_int)
             # Decode loop.
-            # TODO: Use correct eot token from config.
             exec.start_position = len(self.input_token_ids) - 1
-            while token_int != 128001:
+            # TODO: Use correct eot token from config.
+            # while token_int != 128001:
+            for i in range(15):
```

Comment:
needs a proper fix to iterate over a variable number of tokens. okay to land and iterate.